data_model/src/query/cursor.rs

```diff
@@ -27,6 +34,12 @@ pub mod model {
         /// Pointer to the next element in the result set
         pub cursor: Option<NonZeroU64>,
     }
+
+    impl ForwardCursor {
+        pub const fn new(query_id: Option<String>, cursor: Option<NonZeroU64>) -> Self {
```

Comment:
you can use `derive_more::Constructor`

Reply:
At first look yes, but in fact I need this `const`, which `derive_more::Cosntructor` cannot provide. I left a comment for this function

Reply:
comment is not required because anyone who tries to change it will find the reason. Also, I know it `derive_more::Constructor` doesn't make `const` functions